src/components/PublishingTopbar/PublishingTopbar.tsx

```diff
@@ -103,7 +104,19 @@ const PublishingTopbar: React.FC = () => {
       <StyledTopbarBase variant="studio">
         <StudioContainer>
```

Comment:
I know that is not related to this task, but I really don't like that name. We have already `StudioContainer` and it's responsible for studio layout.  I wrongly imported this guy once 😅 
Could you rename this component in this PR?